Advent_2a.py

```diff
@@ -0,0 +1,14 @@
+f = open ( 'input.txt' , 'r')
```

Comment:
`f = open('input.txt', 'r')` is the typical formatting. See https://www.python.org/dev/peps/pep-0008/

Reply:
You should also close files when you are done. This is typically done by using a `with` statement. See https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files